projects/Mallard/src/helpers/issues.ts

```diff
@@ -44,3 +44,8 @@ export const useIssueDate = (issue?: Issue): IssueDate =>
         () => (issue ? renderIssueDate(issue.date) : { date: '', weekday: '' }),
         [issue && issue.key, issue],
     )
+
+export const todayAsFolder = (): string  => {
+    const today = new Date();
```

Comment:
How will this work if a user is in say Australia?

Reply:
Answer is on the next line - getting the UTC date of the current date. So it should be timezone independent.

Reply:
Sorry missed that - thanks.